### PR TITLE
Create Major Release Process.md

### PR DESCRIPTION
**Pull request by tonybalandiuk**
Requesting a review on the content that is there in this initial release process. Provided there are no concerns with the content that's in there, I would like to get this merged, then address TODOs in future PRs.

The intent of this doc is to outline the steps in our Major Release Process as it exists today. It's out starting point/baseline.
Any improvements to the process should be addressed in new requests/commits. 

Signed-off-by: Tony Balandiuk <85971417+tonybalandiuk@users.noreply.github.com>

### CODE REVIEW

File: releases/Process/Major Release Process.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+
+## Terminology
+*Intent: Define terms used throughout the release process* 
+
+**Emergency Release:**  also known as a “hotfix”, an emergency updates to fix a specific set of bugs deemed as either critical security issues or blocking basic utilization of the project. Emergency releases result in an incremental update of the version number (i.e. 22.05.0 would become 25.05.1).
+
+**Exception/Exception Process:** The process by which a SIG requests that a particular pull request be introduced into the Stabilization branch during the "soft lock" phase of the stabilization process.
+
+**Feature List:** A list of all of the features in a release. For O3DE, feature lists consist of a summary section at the top with a summary of the main purpose of the release and feature highlights. Then, below this section, the features are listed, grouped by SIG. The Feature list has 2 purposes. 1) it provides the community and marketing with a summary of what to expect in the release 2) it is used by the Documentation Program Manager to create the release notes.
```
Comment (vincent6767) [inline releases/Process/Major Release Process.md:45]: What are your thoughts on adding the example of the features list done in the past? @tonybalandiuk

Comment (vincent6767) [reply releases/Process/Major Release Process.md:45]: Nevermind. You have the example in the 1.2 section.

**Review by vincent6767 — APPROVED** [review]
Looks good to me for the initial content. I suggest having a status as "Draft" in the document so people know it's still in progress.

File: releases/Process/Major Release Process.md (hunk 1 above)
Comment (tonybalandiuk) [reply releases/Process/Major Release Process.md:45]: good catch @vincent6767  - I added "Draft" to the title

**Pull request merged**